extensions/applicationinsights-clickanalytics-js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@microsoft/applicationinsights-clickanalytics-js",
-  "version": "2.5.10",
+  "version": "2.5.11",
```

Comment:
We don't update this separately, we update all of the versions are part of the Release PR

Reply:
Reverted to 2.5.10